src/lib/es2018.intl.d.ts

```diff
@@ -33,9 +33,16 @@ declare namespace Intl {
     const PluralRules: {
         new (locales?: string | string[], options?: PluralRulesOptions): PluralRules;
         (locales?: string | string[], options?: PluralRulesOptions): PluralRules;
-        supportedLocalesOf(
-            locales: string | string[],
-            options?: PluralRulesOptions,
-        ): string[];
+
+        supportedLocalesOf(locales: string | string[], options?: { localeMatcher?: "lookup" | "best fit" }): string[];
     };
+
+    interface NumberFormatPart {
+        type: "compact" | "currency" | "decimal" | "exponentInteger" | "exponentMinusSign" | "exponentSeparator" | "fraction" | "group" | "infinity" | "integer" | "literal" | "minusSign" | "nan" | "plusSign" | "percentSign" | "unit" | "unknown";

```

Comment:
This seems to be the es2020 set, es2018 parts don't have `compact | exponentInteger | exponentMinusSign | exponentSeparator | unit | unknown`. 

- `compact` comes from compact notation which is es2020
- `exponentInteger | exponentMinusSign | exponentSeparator` comes from scientific notation in es2020
- `unit` comes from unit formatting in es2020
- `unknown` is just `literal`.

This is missing `symbol` from `currency` formatting as well.

Overall I think the set for es2018 is `literal | nan | infinity | percent | integer | group | decimal | fraction | plusSign | minusSign | percentSign | currency | code | symbol | name`

Reference: https://402.ecma-international.org/5.0/#sec-partitionnumberpattern